Objective-C/Symbol Index.tmPreferences

```diff
@@ -2,11 +2,13 @@
 <plist version="1.0">
 <dict>
 	<key>scope</key>
-	<string>source.objc, source.objc++</string>
```

Comment:
This file I also stumbled over.

It is originally meant to do some symbol translations only. Its scopes overlap with the same file in C++ package and the translation rule conflicting with the other file. Haven't looked into it any further though.

But as _C++/Symbol List.tmPreferences_ already adds `entity.name.constant` to symbol list and index for `source.objc` and `source.objc++` your change here is not needed.

Reply:
Interesting. I didn't see the `source.objc` and `source.objc++`. So to be clear, revert this file but keep the `Symbol Index - Include Constants.tmPreference` deleted? Seeing as how the `entity.name.constant` is already added via ` C++/Symbol List.tmPreferences`

Reply:
Complete